src/main/java/club/issizler/okyanus/api/cmd/CommandRunnable.java

```diff
@@ -1,16 +1,9 @@
 package club.issizler.okyanus.api.cmd;
 
-/**
- * Your should either
- * - Implement this in a class
- * - Create an anonymous class implementing this
- * <p>
- * And pass it to {@link CommandBuilder#run(CommandRunnable)} to make your
- * command actually do something
- */
+@Deprecated
```

Comment:
You're deprecating CommandRunnable, but what alternative are you providing? Even in runtime, you seem to keep using it.

Reply:
I see that you just moved the class. You should keep using this one instead of having duplicate code, as we'll eventually merge both `cmd` and `cmdnew` into a single package on 1.0.0.